src/pages/cheerpj-jnlprunner/index.astro

```diff
@@ -0,0 +1,194 @@
+			<div
+				class="mb-9 rounded-xl py-8 px-7 shadow-md transition-all hover:shadow-lg sm:p-9 lg:px-6 xl:px-9"
+			>
+				<div class="mx-auto mb-7 inline-block">
+					<Image src={billIcon} width="53" height="61" alt="bill icon" />
+				</div>
+				<div>
+					<h3
+						class="mb-4 text-xl font-bold text-white sm:text-2xl lg:text-xl xl:text-2xl"
+					>
+						Cost-effective
+					</h3>
+					<p class="text-base font-medium text-body-color">
+						The JNLP-Runner extensions is available for a very affordable price.
+						It is an option for both companies and individual users.
+					</p>
+				</div>
+			</div>
+		</div>
+
+		<div class="w-full px-4 md:w-1/2 lg:w-1/3">
+			<div
+				class="mb-9 rounded-xl py-8 px-7 shadow-md transition-all hover:shadow-lg sm:p-9 lg:px-6 xl:px-9"
+			>
+				<div class="mx-auto mb-7 inline-block">
+					<Image src={layersIcon} width="53" height="61" alt="layers icon" />
+				</div>
+				<div>
+					<h3
+						class="mb-4 text-xl font-bold text-white sm:text-2xl lg:text-xl xl:text-2xl"
+					>
+						Wide range of utilities
+					</h3>
+					<p class="text-base font-medium text-body-color">
+						CheerpJ JNLP-Runner extension cover all your Java needs, including
+						Swing/AWT, filesystem support, audio, clipboard, networking and
+						more.
+					</p>
+				</div>
+			</div>
+		</div>
+
+		<div class="w-full px-4 md:w-1/2 lg:w-1/3">
+			<div
+				class="mb-9 rounded-xl py-8 px-7 shadow-md transition-all hover:shadow-lg sm:p-9 lg:px-6 xl:px-9"
+			>
+				<div class="mx-auto mb-7 inline-block">
+					<Image src={rocketIcon} width="53" height="61" alt="rocket icon" />
+				</div>
+				<div>
+					<h3
+						class="mb-4 text-xl font-bold text-white sm:text-2xl lg:text-xl xl:text-2xl"
+					>
+						Easy to use
+					</h3>
+					<p class="text-base font-medium text-body-color">
+						Using our browser extension could not be easier. Just install,
+						launch your JNLP app as usual and the extension will automatically
+						start working.
+					</p>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div
+		class="px-4 sm:px-8 py-16 space-y-4 mx-4 flex col-span-2 items-center justify-center p-10"
+	>
+		<div>
+			<h3
+				class="font-bold text-right text-3xl leading-[1.1] sm:text-3xl md:text-6xl"
+			>
+				The <span class="font-bold text-primary-500"> JNLP</span> extension for
+				the browser
+			</h3>
+			<p class="text-right">
+				Follow our channels to stay up to date with our community.
+			</p>
+			<div class="mt-8 flex flex-wrap justify-end gap-4 pt-10">
+				<LinkButton
+					href="https://github.com/leaningtech/cheerpj-jnlprunner"
+					label="Report a bug"
+					type="primary"
+					iconRight="mdi:github"
+				/>
+				<LinkButton
+					href={DISCORD_URL}
+					label="Join the community"
+					type="discord"
+					iconRight="ic:baseline-discord"
+				/>
+			</div>
+		</div>
+		<div></div>
+		<div class="flex justify-center items-center">
+			<Image class="scale-75" src={screenshots} alt="screenshots" />
```

Comment:
It'd be really nice to incorporate screenshots of the extension into the hero. I think how I did it with my side project Mamar is a nice way of doing it: see https://mamar.nanaian.town/

Reply:
Added :)